pkgs/tools/security/otpgen/default.nix

```diff
@@ -0,0 +1,40 @@
+{ 
+  pkgs
+  , lib
+  , stdenv
+  , makeWrapper
+  , fetchFromGitHub
+  , cmake
+  , pkg-config
+  , cryptopp
+  , qtbase
+}:
+
+stdenv.mkDerivation {
+
+  pname = "OTPGen";
+  version = "0.9.3";
+
+  meta = {
```

Comment:
Nixpkgs has a conventional ordering of attributes, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Fixed here https://github.com/NixOS/nixpkgs/pull/87887/commits/9aefdb5d5ba41f89b1d57a6c538ec16a25fffe03
Unsure of uncovered attribute depsBuildBuild, choosed to place it between nativeBuildInputs and buildInputs meanwhile.